graphene_sqlalchemy/tests/models_batching.py

```diff
@@ -60,9 +62,14 @@ class Reporter(Base):
     articles = relationship("Article", backref="reporter")
     favorite_article = relationship("Article", uselist=False)
 
-    column_prop = column_property(
-        select([func.cast(func.count(id), Integer)]), doc="Column property"
-    )
+    if SQL_VERSION_HIGHER_EQUAL_THAN_1_4:
+        column_prop = column_property(
+            select(func.cast(func.count(id), Integer)), doc="Column property"
+        )
+    else:
+        column_prop = column_property(
+            select([func.cast(func.count(id), Integer)]), doc="Column property"
```

Comment:
Actually as you are doing the `select(...)` vs. `select([...])` a lot, maybe a helper function:
```python
def wrap_select_queries(query):
    if SQL_VERSION_HIGHER_EQUAL_THAN_1_4:
        return select(query)
    else:
        return select([query])
```
might be useful 🙂 
Makes dropping the <1.4 support later a lot easier as you would just need to search for `wrap_select_queries` and replace it with select 🙂